src/hooks/wallets/useOnboard.test.ts

```diff
@@ -45,4 +45,24 @@ describe('getConnectedWallet', () => {
       chainId: '4',
     })
   })
+
+  it('Failed to get connected wallet', () => {
```

Comment:
We tend to describe the expectation of a test in it's name.

```suggestion
  it('should return null if the address is invalid', () => {
```